lessons/Early-Math/variables/langs/en-us/pages/variables-code2.adoc

```diff
@@ -0,0 +1,71 @@
+= Variables and Code (Associative Property)
+
+@n Open @starter-file{variables-code2}. On the table below, record your Noticings and Wonderings about what you see there. Consider how this starter file is different from @starter-file{variables-code}. *Don't hit "Run" yet!*
+
+[.FillVerticalSpace,cols="1, 1", stripes="none", options="header"]
+|===
+
+| Notice | Wonder
+|
+|
+
+|===
+
+@n Based on what you see in the Definitions Window (left side), predict what will happen when you hit "Run" by circling your choice below.
+
+@vspace{1ex}
+
+[cols="^.^1,^.^1,^.^1,^.^1", stripes="none"]
+|===
+
+| Looks shipshape, all 4 tests passed, mate!
```

Comment:
The solutions version of this question does not show an answer

Reply:
That's because students are making a prediction!

Reply:
oh yeeeeah! Sorry about that :P